src/main/main.ts

```diff
@@ -246,7 +241,21 @@ const openPathDialog = (event: any, args: any) => {
     console.log(err);
   })
 } 
+/**
+ * Checks the app config.
+ * @returns true if config is setup, false otherwise. 
+ */
+const checkConfig = () : boolean => {
+  return mainWindow !== null ? isConfigReady(cfg) && (getLatestLog(baseLogPath)) : false;
```

Comment:
I think we wanted rid of the getLatestLog() check here?

Can we remove it? I'm not 100% sure, need to check if watchLogs will do the right thing which I think is nothing till a log is created then. 

A test case would be to move all your logs out the folder and see what happens when you drop one back in with the app running.

Reply:
Also I think parenthesis could be neater as:

`return mainWindow !== null ? (isConfigReady(cfg) && getLatestLog(baseLogPath)) : false;`

Reply:
I am going to be removing all of those latest logs checks in a later PR when i work on the no logs in the directory issue, didnt want to clutter this reivvew. 

Also, i actually disagree on the parenthesis, only because if we ever get a linter it will throw a fit with unnecessary parenthesis. -- i know this because our work linter does this for this exact code lol

Reply:
Hmm OK - in that case can remove the parenthesis around `(getLatestLog(baseLogPath))` ? 

Whatever, not that big a deal, not gonna let it block merging.